/**:
  ros__parameters:

    start_speed: 0.50   # Начальная максимальная скорость робота
    start_offset: -70.0 # Начальное смещение от центра полосы

    # Размера буфера с ошибками
    max_len: 50 

    # PID константы для угла поворота  
    Kp_ang: 0.004
    Ki_ang: 0.00003
    Kd_ang: 0.0

    # PID константы для скорости
    Kp_vel: 0.0008
    Ki_vel: 0.000005
    Kd_vel: 0.0

    # Минимальная площадь bbox для детекции знака
    min_square: 6000

    # Количество детекций для усреднения ответа
    detects_cnt: 5